src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -76,8 +76,6 @@ public SwerveModule(CANSparkMax driveMotor, CANSparkMax angleMotor, CANCoder can
 
         m_driveEncoder.setVelocityConversionFactor(SwerveDriveModuleConstants.k_CANEncoderVelocityCoefficient);
         m_driveEncoder.setPositionConversionFactor(SwerveDriveModuleConstants.k_CANEncoderPositionCoefficient);
-    
-      System.out.println("her!!!e");
```

Comment:
Just making a comment for the practice of it.

Reply:
👍